src/components/header/header.css

```diff
@@ -0,0 +1,46 @@
+body{
```

Comment:
Why is the a body rule in this file?

Reply:
To solve a margins problem with the header.

Reply:
Body rules should be in the main.css file, this file is only for the header